MANIFEST.in

```diff
@@ -1,3 +1,3 @@
 include versioneer.py
 include napari_gui/_version.py
-recursive-include napari_gui/elements/qt/icons
+recursive-include napari_gui/icons
```

Comment:
We should use a more generic path like `res` (for resources) and add an `icons` subdirectory.

Reply:
Added one called resources and put icons in a subdirectory. I thought `res` was a little vague, but happy to change